### PR TITLE
P14C-8089: Update pingone-customers-sample-resource-owner sample to support https

### PR DESCRIPTION
**Pull request by deivur**
Updated spring dependencies and added https support.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,11 +11,11 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.1.1.RELEASE</version>
+    <version>2.2.4.RELEASE</version>
   </parent>
 
   <properties>
-    <spring-boot.version>2.1.1.RELEASE</spring-boot.version>
+    <spring-boot.version>${parent.version}</spring-boot.version>
```
Comment (hkrop) [inline pom.xml]: Can't find this property (it's [not the default one too](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Project_Interpolation))

**Review by hkrop — CHANGES_REQUESTED** [review]
Looks good, with minor questions

File: resource-server-sample/pom.xml [hunk 2]
```diff
@@ -4,18 +4,19 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.1.RELEASE</version>
+        <groupId>com.pingidentity.samples</groupId>
```
[the remark this answers is not in the document]
Comment (Deivur) [reply resource-server-sample/pom.xml]: I see, removed.

File: pom.xml (hunk 1 above)
Comment (Deivur) [reply pom.xml]: Got it, thank you!

**Comment by Deivur** [comment]
Сorrected all comments.

**Pull request merged**